app/views/groups/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Prefrences for Group <%= @group.group_name %></h1>
+
+<%= form_tag("/groups/" + params[:id], method: "put") do %>
+    <h3>First project choice</h3>
+    <p>Currently <%= first_choice %></p>
+    <select name="first_choice">
+        <% @projects.each do |project| %>
+            <option value=<%= project[:project_name] %>><%= project[:project_name] %></option>
+        <% end %>
+    </select>
+    
+    <h3>Second project choice</h3>
+    <p>Currently <%= second_choice %></p>
+    <select name="second_choice">
+        <% @projects.each do |project| %>
+            <option value=<%= project[:project_name] %>><%= project[:project_name] %></option>
+        <% end %>
+    </select>
+    
+    <h3>Third project choice</h3>
+    <p>Currently <%= third_choice %></p>
+    <select name="third_choice">
+        <% @projects.each do |project| %>
+            <option value=<%= project[:project_name] %>><%= project[:project_name] %></option>
+        <% end %>
+    </select>
+    <br>
```

Comment:
I'm positive we can get a for loop here to populate a table view instead of hard coding each preference one by one. It'll be easier to come back and add UI elements later if it's just a table view too